### DISCUSSION TITLE
Moving entire canvas using right click and drag

### DISCUSSION THREAD

**Discussion by rdc3-philips** [excalidraw/excalidraw-vscode#120] [Ideas]
If we can move the entire canvas with right click + drag, it would reduce a mouse click on the hand icon. 
It might just be a simple change in code if the hand icon logic could be reused. But the shortcut would be a drastic change in terms of usage.
1 additional click ( on the hand icon) to drag the entire screen is annoying if you are editing a big canvas and you need to constantly move from one end to another.
Can you please add this feature.

**Comment by derSebastian** [comment]
Just FYI, you can drag the canvas by holding down the spacebar and just dragging with your left mouse button.
